aries-site/src/scripts/start-tests.js

```diff
@@ -0,0 +1,59 @@
+const createTestCafe = require('testcafe');
+const BrowserStack = require('browserstack');
+
+const browserStackCredentials = {
+  username: process.env.BROWSERSTACK_USERNAME,
+  password: process.env.BROWSERSTACK_PASSWORD,
+};
+
+// TBD: What browsers to we want to test against
+const SUPPORTED_BROWSERS = ['browserstack:firefox@72.0:Windows 10'];
```

Comment:
Can we add more browsers and use other OS on the same runs?
I'm thinking Chrome on both WIN and MAC is a must.

Reply:
Yes, we should definitely add more. I just did one for now when I was checking to make sure that it was connecting to the account correctly.

Reply:
We'll probably need to find the right line on how many combinations of OS+browsers we should support without effecting the build time too bad. will be interesting to see how it grows.

Reply:
I think if the majority of us have Chrome on our desktop we shouldn't use it in the Supported Browsers on the browser stack I think if we are limited on min/month we should test the browsers that we don't have like ie and edge and maybe Firefox but I think most of us have chrome and safari on our local computer. 

Something to discuss in the future as we get closer to that min/month limit